python/setup.py

```diff
@@ -192,6 +192,9 @@
       ext_modules=[crypto_module, key_value_module],
       entry_points = {
           'console_scripts': [
+              'pdo-configure-services = pdo.scripts.ConfigureCLI:configure_services',
+              'pdo-configure-users = pdo.scripts.ConfigureCLI:configure_users',
+              'pdo-configure-ccf = pdo.scripts.ConfigureCLI:configure_ccf',
```

Comment:
Why not `pdo-configure-ledger`?

Reply:
because, in this case, it really is unique to ccf. as i've said... i would really prefer that ccf have its own parallel configuration script. this one feels a bit out of place.